### PR TITLE
Sudo remove, quest changes, achievement add, rating in diff

### PR DESCRIPTION
**Pull request by Mj11jM**
Added a new quest and achievement

Fixed wording on tavern, as 1 quest is default

Changed tag list to show 100 per page and it properly removes the removed tags from displaying

Reduced rewards on a lot of quests and tweaked the rewards as well, this is not set, we will see how these changes affect things.

### CODE REVIEW

**Comment by Mj11jM** [comment]
I've done 2 more commits, these fix the fraud report 3 grabbing non-sold auctions, and a fix for the passive slot showing empty while still believing there is something equipped. All tested as functional  on my test bot

File: commands/hero.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ const Anilist = new anilist();
 
 cmd(['hero'], withUserEffects(async (ctx, user, effects, ...args) => {
     const now = new Date()
+    await user.save()
```
Comment (NoxCaos) [inline commands/hero.js:42]: That's interesting, but I guess it works.

**Review by NoxCaos — APPROVED** [review]

File: commands/hero.js (hunk 1 above)
Comment (Mj11jM) [reply commands/hero.js:42]: So far this only does and saves the changes on checking those slots. hence why that save is only there and in ->slots. I can move it to with user effects. but this is less saving and we know there is nothing else being edited with the user at this time

Comment (NoxCaos) [reply commands/hero.js:42]: I think it could be even put into `withUserEffects`

**Pull request merged**